pages/api/admin/getRegistrations.test.ts

```diff
@@ -0,0 +1,46 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { initDB } from "../../../lib/db/init";
+import { register } from "../../../lib/api/users/user"; 
+import handler from "./getRegistrations";
+import { ECS_ID, ELW_ID } from "../../../lib/locker_constants";
+
+
+const mockedJson = jest.fn();
+
+const mockedRes = {
+    status: jest.fn().mockReturnValue({ json: mockedJson }),
+} as unknown as jest.Mocked<NextApiResponse>;
+
+initDB();
```

Comment:
rather than using `initDB()` once, we should use it in a BeforeEach block to eliminate dependency between tests